### PR TITLE
Int 6679

### PR DESCRIPTION
**Pull request by sl45082**
This PR for update to sdk for shrinkBatchRawData() by upgrading to latest sdk in package.json.
J1/Schema/User entity added an "active" property which drove the changes for the files outside of package.json. The "active" property is assigned a value from Onelogin based on: https://developers.onelogin.com/api-docs/1/users/user-resource

### CODE REVIEW

File: src/converters/UserEntityConverter.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ export function createUserEntity(
     ...convertProperties(user.custom_attributes, {
       prefix: 'customAttributes', //used to be custom_attributes
     }),
+    active: user.status === 1,
```
Comment (dobregons) [inline src/converters/UserEntityConverter.ts:75]: It'd be nice to create an enum that matches the different statuses in onelogin based on the link that you provided, something like:
```
export enum OneLoginUserStatus {
  UNACTIVATED = 0,
  ACTIVE = 1,
  SUSPENDED = 2,
  LOCKED = 3,
  PASSWORD_EXPIRED = 4,
  AWAITING_PASSWORD_RESET = 5,
  PASSWORD_PENDING = 7,
  SECURITY_QUESTIONS_REQUIRED = 8
}
```

and here we'd have a more readable code:
```
active: user.status === OneLoginUserStatus.ACTIVE
```

**Review by dobregons — COMMENTED** [review]
In addition to that, please don't forget to update the [changelog.md](https://github.com/JupiterOne/graph-onelogin/blob/main/CHANGELOG.md).

File: src/converters/UserEntityConverter.ts (hunk 1 above)
Comment (sl45082) [reply src/converters/UserEntityConverter.ts:75]: thanks for feedback. will update and bring into branch.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ and this project adheres to
 
 ## [Unreleased]
 
+## 2.3.1 - 2023-02-03
+
+### Fixed
+
+- Updated package.json to latest sdk, to fix the r shrinkBatchRawData() errors
+  Latest sdk uses an updated J1/schema/User entity so changes were requred in
```
Comment (dobregons) [inline CHANGELOG.md:16]: small typo required -> requred

**Review by dobregons — APPROVED** [review]

**Review by dobregons — APPROVED** [review]

**Pull request merged**